server/monitor/src/main/java/org/apache/accumulo/monitor/next/InformationFetcher.java

```diff
@@ -236,9 +236,9 @@ public void newConnectionEvent() {
   }
 
   // Protect against NPE and wait for initial data gathering
-  public SystemInformation getSummary() {
+  public SystemInformation getSummary() throws InterruptedException {
     while (summaryRef.get() == null) {
-      Thread.onSpinWait();
+      Thread.sleep(100);
```

Comment:
It seems like it would be better to handle InterruptedException here, rather than throw it out and handle it everywhere else.